### ISSUE TITLE
GOTTSCHALK source description has weird spacing

### ISSUE THREAD

**Issue by dchiller** [DDMAL/CantusDB#1219]
Source detail page here: https://cantusdatabase.org/source/123609

I can't get it in a screenshot at normal sizing, but when I zoom out, you can see that there is a table that gets pushed all the way down the page: 

<img width="1321" alt="image" src="https://github.com/DDMAL/CantusDB/assets/11023634/2af9fcd4-cffc-4843-a5c9-c02d3ee11c7a">

**Comment by dchiller** [comment]
In the html, there are many many `<br>` tags between the colon and the table definition. When those are manually removed, the spacing is normal. I suspect those are being added automatically somehow.

There is a single `<br>` tag at this point in the Source Description field, but removing it does not seem to fix the problem.

**Comment by annamorphism** [comment]
Is that what it is! I've been annoyed at this table before...

**Comment by jacobdgm** [comment]
This was introduced by #1087 - when I remove `linebreaks` locally, the extra spaces disappear.

I might suggest that we get rid of the `|linebreaks` "decorator" everywhere on the Source Detail page - it's causing this here problem, and it's causing problems in the Bibliography field (#1221). We may need to tell the musicologists to go through the text fields for all the sources and format everything via html tags, but we can't accommodate both text spaced out with linebreaks and text spaced out by html tags.

**Comment by annamorphism** [comment]
but OldCantus seemed to handle this situation...
https://cantus.uwaterloo.ca/node/123609/edit
<img width="426" alt="image" src="https://github.com/DDMAL/CantusDB/assets/67451875/67bbfca7-a2b2-4d25-8103-69a568c61629">
rendered as 
<img width="598" alt="image" src="https://github.com/DDMAL/CantusDB/assets/67451875/817f7a34-c234-4172-b8d9-8e37de1706e4">

while this also works fine:
<img width="315" alt="image" src="https://github.com/DDMAL/CantusDB/assets/67451875/e77fc96c-3edd-4b96-8574-253e311c993a">
<img width="584" alt="image" src="https://github.com/DDMAL/CantusDB/assets/67451875/1e234902-8ddf-48d1-80b7-6ceb9c942f12">

**Comment by dchiller** [comment]
> We may need to tell the musicologists to go through the text fields for all the sources and format everything via html tags, but we can't accommodate both text spaced out with linebreaks and text spaced out by html tags.

Or do this automatically, no?

**Comment by annamorphism** [comment]
combining different breaks seems to be handled slightly differently from the old site. here's oldcantus (https://cantus.uwaterloo.ca/node/715445/edit):
<img width="457" alt="image" src="https://github.com/DDMAL/CantusDB/assets/67451875/9cf693f9-7a86-4238-a803-9e378dda6602">

which renders like so: 
<img width="276" alt="image" src="https://github.com/DDMAL/CantusDB/assets/67451875/1f7c08c2-4ec6-42dc-8024-2523d5a1b57a">
while the same on NewCantus renders like so (https://cantusdatabase.org/source/1000016):
<img width="285" alt="image" src="https://github.com/DDMAL/CantusDB/assets/67451875/74a506f4-2fe7-4e33-aba2-f9808bf4a9f3">

it looks like OldCantus made a big space after the break tag and treated all the other tags identically, while NewCantus styles the paragraph with more space after it.

**Comment by jacobdgm** [comment]
Those unclosed `<p>`s are making my head hurt... are there sources where this is done?

**Comment by jacobdgm** [comment]
> > We may need to tell the musicologists to go through the text fields for all the sources and format everything via html tags, but we can't accommodate both text spaced out with linebreaks and text spaced out by html tags.
> 
> Or do this automatically, no?

What kind of "automatically" do you mean? We write a script to replace each `\n` in the relevant fields with a `<br>`?

**Comment by annamorphism** [comment]
> Those unclosed `<p>`s are making my head hurt... are there sources where this is done?

I don't think so--I initially wrote it in error, and then wondered if such mistakes had any impact on the end result (they don't seem to?)

**Comment by jacobdgm** [comment]
Having thought about this for a while, here's an approach we might take:

We check to see if the bibliography (we can take a similar approach with description, etc.) contains any html tags. If it does, we assume that any linebreaks are there to make the html easy to read, so we display it without added linebreaks. If there are no html tags, we assume that any line breaks were introduced to format the text in the field, so we add the `linebreaks` decorator.

UPDATE: Having discussed this at lab meeting, we're going to try this approach, and then examine a large sample of sources to make sure there are no significant errors. If we find significant errors, we'll roll back the change and get someone to manually reformat them Summer 2024.

**Comment by annamorphism** [comment]
I came up with something similar independently, so I am in favour!

**Comment by jacobdgm** [comment]
Noting that this issue is similar to #1221 and #1239

**Comment by liampond** [comment]
July 22 meeting: we would like to remove paragraph indentation
